tests/validate_plusless_properties.py

```diff
@@ -22,13 +24,14 @@ def check_equivalency(key, node_type, node_properties):
         difference = diff(key_properties, counterpart_properties)
         raise Exception(f"{key} and {counterpart_key} both exist in {node_type}, but are different: {difference}")
 
-node_types = ['model_configs', 'seed_configs', 'snapshot_configs', 'source_configs', 'test_configs']
-for filepath in PROJECT_SCHEMA_FILES:
-    with open(filepath, "r") as f:
+node_types = ['model_configs', 'seed_configs', 'snapshot_configs', 'test_configs']
```

Comment:
looks like we dropped `source_configs` here -- was that intentional?

Reply:
Yes, since I'm not sure if the groups can be defined for `source_configs` inside the `dbt_project` files. There is no info about `source_configs` on the [Groups docs page](https://docs.getdbt.com/reference/resource-configs/group).
![image](https://github.com/dbt-labs/dbt-jsonschema/assets/7255359/e5d75680-6f5e-4d1c-86b2-ae27867af747)

Reply:
you're right that those can't be grouped, but we definitely want to keep testing them!

Reply:
I think this test _shouldn't_ need to be edited here IIRC!

Reply:
There should be at least [this line](https://github.com/dbt-labs/dbt-jsonschema/pull/94/files#diff-dc2ed06cdc0f621f65990807cea4a080bb690a4b9cb5e91fe1cb151882a9667dL14) fix, otherwise the tests fail

Reply:
Just wondering if a source is a node type(?), then groups is listed as a possible but optional property at least in the model.  But definitely not the core expert here, so I'm sure I'm missing a ton of background, and maybe it's an unsupported (but still there) property?

https://github.com/dbt-labs/dbt-core/blob/ef9d6a870f3cc864ba3240d393a1a9a5b1de529d/core/dbt/contracts/graph/nodes.py#L336

Reply:
> I think this test shouldn't need to be edited here IIRC!
I agree, I reverted tests changes and left only configs changes here.

Reply:
@dave-connors-3 is here anything which still needs to be resolved?

Reply:
@evgeniybobok no I think it's all good!